pkg/agent/controller/networkpolicy/audit_logging.go

```diff
@@ -127,7 +133,7 @@ func (l *AntreaPolicyLogger) updateLogKey(logMsg string, bufferLength time.Durat
 // LogDedupPacket logs information in ob based on disposition and duplication conditions.
 func (l *AntreaPolicyLogger) LogDedupPacket(ob *logInfo) {
 	// Deduplicate non-Allow packet log.
-	logMsg := fmt.Sprintf("%s %s %s %s %s %s %d %s", ob.tableName, ob.npRef, ob.disposition, ob.ofPriority, ob.srcIP, ob.destIP, ob.pktLength, ob.protocolStr)
+	logMsg := fmt.Sprintf("%s %s %s %s %s %s %s %s %s %d", ob.tableName, ob.npRef, ob.disposition, ob.ofPriority, ob.srcIP, ob.srcPort, ob.destIP, ob.destPort, ob.protocolStr, ob.pktLength)
```

Comment:
If this is ICMP, the number of fields will be different from TCP/UDP? To avoid parsing issue, could we have some placeholder for ICMP to keep the format consistent?

Reply:
Thanks for the reminder, changed to "<nil>" as a placeholder, to have a consistent result in case IP is nil for `IP.String()`. Let me know if there's a better placeholder.

Reply:
The placeholder is not displayed in your comment but I get your mean `<nil>`, which looks good to me.

Reply:
Updated comment at line 232 where `<nil>` was assigned.